src/app/downloads/downloads.component.css

```diff
@@ -5,6 +5,7 @@
 }
 .controls {
   flex: 1 1 20%;
+  max-width: 20%;
```

Comment:
Can we change this to 30/70 so it doesn't look so terrible in the meantime?
`.controls {
  flex: 1 1 30%;
  max-width: 30%;
}
.content {
  flex: 7 7 70%;
  max-width: 70%;
}`